plugins/main/public/components/common/ribbon/ribbon-item.tsx

```diff
@@ -51,9 +54,7 @@ interface RibbonItemProps {
 const WzRibbonItem = (props: RibbonItemProps) => {
   const { item } = props;
 
-  const elementStyle = { ...(item.style || {}), fontSize: FONT_SIZE };
-  const wzWidth100 = { anchorClassName: 'wz-width-100' };
-  const tooltipProps = item.key === 'cluster-node' ? wzWidth100 : {};
+  item.style = { ...item.style, fontSize: FONT_SIZE };
```

Comment:
**nitpick**: this seems to redefine a prop and this is a bad practice. Create a new object/variable to append the `fontSize` definition.